docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -84,118 +90,170 @@ Golden Signals are a form of telemetry that applies to anything with throughput.
 
 These golden signals are closely related to the <i>[RED metrics](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture/)</i> for microservices: rate, errors, and duration, and the older <i>[USE method](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture/)</i> focusing on utilization, saturation, and errors. These signals are used to calculate the service level objectives (SLOs).
 
-Here are other useful metrics that are examples which don’t necessarily fit into the Golden Signals.
-
-- Pull Request: Time to Merge
-
-- Pull Request: Lead Time
-
-- Pull Request: Size (LOC)
+### Other Signals
+Here are other useful metrics that are examples that don’t necessarily fit into the Golden Signals.
 
-- Pull Request: Flow Ratio - the sum of the opened pull request in a day divided by the sum of closed pull requests in that same day. This metric shows whether your team works in a healthy proportion.
+---
+#### Pull Requests
+
+  <dl>
+    <dt>Time to Merge</dt>
+    <dd>The duration between the creation of a pull request and when it's merged. This metric reflects the efficiency of
+      the review and merging process.</dd>
+    <dt>Lead Time</dt>
+    <dd>The total time from when work on a pull request starts until it's merged. It measures the overall speed of the
+      development process.</dd>
+    <dt>Size (LOC)</dt>
+    <dd>The number of lines of code (LOC) changed in a pull request. This metric helps gauge the complexity and potential
+      impact of the changes.</dd>
+    <dt>Flow Ratio</dt>
+    <dd>The ratio of the total number of pull requests opened to those closed in a day. It indicates whether the team's
+      workflow is balanced and sustainable.</dd>
+    <dt>Discussions & Comments</dt>
+    <dd>The number of comments and discussions on a pull request. This metric shows the level of collaboration and code
+      review quality.</dd>
+    <dt>Force Merged by Admins (bypassing approvals)</dt>
+    <dd>The number of pull requests merged by admins without the usual approval process. This metric highlights instances
+      where standard review procedures were skipped.</dd>
+  </dl>
+---
+#### Code Quality
+
+    <dl>
+      <dt>Code Coverage</dt>
+      <dd>The percentage of your codebase covered by automated tests. This metric helps ensure that your code is thoroughly
+        tested.</dd>
+      <dt>Static Code Analysis</dt>
+      <dd>The number of issues found by static code analysis tools. This metric helps identify potential bugs and security
+        vulnerabilities.</dd>
+      <dt>Defect Escape Rate</dt>
+      <dd>The percentage of defects found in production that weren't caught by automated tests. This metric indicates the
+        effectiveness of your testing strategy.</dd>
+    </dl>
 
-- Pull Request: Discussions # Comments
+---
```

Comment:
Seems like we're introducing more styling, and a convention of lines above `####`. We can do that, but it should be a CSS decision. 
```suggestion
```